pkg/models/plan_cr.go

```diff
@@ -24,3 +26,36 @@ func NewPlanCR(createdBy uuid.UUID, modelPlanID uuid.UUID) *PlanCR {
 		modelPlanRelation: NewModelPlanRelation(modelPlanID),
 	}
 }
+
+func (cr *PlanCR) ToEchimpCR(associatedModelUids *uuid.UUID) *EChimpCR {
+	// get imp date
+	var implementationDate *string
+	if cr.DateImplemented != nil {
+		implementationDate = helpers.PointerTo(cr.DateImplemented.Format(time.DateOnly))
+	}
+
+	// get cr summary from cr note
+	var crSummary *TaggedContent
+	if cr.Note != nil {
```

Comment:
I think this is just fine as is, but you could use `NewTaggedContentFromString` to remove some boiler place

Reply:
Good to know! Gonna leave it as-is so I don't have to re-test if that's alright 😬

Reply:
That works for me since this is going to be short lived 😄